project/frontend/src/containers/Explanation/Explanation.js

```diff
@@ -23,30 +23,34 @@ const Explanation = (props) => {
         <h1>How does it work?</h1>
         <ol>
           <li>
-            You can find all the competition you’re in in the or create a
-            competition if you don’t have any.
+            You can find all the competitions you're competing in, or create a
+            competition on the competition page.
```

Comment:
*Competition should be capitalised like the other pages

Reply:
fixed 4113c67f69cce5f27b3a2e38d3508adb723ea3b0